### PR TITLE
HIPP-363: Add the accessibility statement to the API Hub

### PR DESCRIPTION
**Pull request by PaulCDurham**
https://jira.tools.tax.service.gov.uk/browse/HIPP-363

### CODE REVIEW

File: conf/services/api-hub.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
+serviceName: The API Hub
+serviceDescription: You can use this service to allow users to discover APIs, consume APIs and manage credentials with self-serve capabilities.
+serviceDomain: admin.tax.service.gov.uk
+serviceUrl: /api-hub
```
Comment (laurahk) [inline conf/services/api-hub.yml:4]: I think this needs changing to 'api-hub-frontend'. I ran up api-hub-frontend and accessibility-statement locally and clicked on the link and it didn't work until I changed the URL.

**Comment by laurahk** [comment]
The statement itself looks great. Just the question about the integration with the frontend.

File: conf/services/api-hub.yml (hunk 1 above)
Comment (PaulCDurham) [reply conf/services/api-hub.yml:4]: It needs to be api-hub. The URL for the service in production is admin.tax.service.gov.uk/api-hub. See the documentation in the README...
serviceUrl: /icecreams                  # The relative URL to the service (omitting www.tax.service.gov.uk)

I think the configuration that might be wrong is this bit in api-hub-frontend's application.conf...

accessibility-statement{
  service-path = "/api-hub-frontend"
}

That should probably change to be "api-hub". That would then match the file name we've created here. The convention in this repo is to use the service name not the frontend name.

Comment (laurahk) [reply conf/services/api-hub.yml:4]: Ah I see. Are you going to put a PR for the frontend now so it works when merged?

Comment (PaulCDurham) [reply conf/services/api-hub.yml:4]: Yup, I'll do that today

**Pull request merged**